ops/internal/manage/staging.go

```diff
@@ -215,3 +222,39 @@ func StagedSuperchainDefinition(rootP string) (*config.SuperchainDefinition, err
 
 	return sM, err
 }
+
+// isChainInDependencySet checks if a chain ID is in the supplied dependency set
+func isChainInDependencySet(chainID common.Hash, depSet *prestate.DependencySet) bool {
+	if depSet == nil {
+		return false
+	}
+	for id := range depSet.Dependencies {
+		if id == chainID.Hex() {
```

Comment:
We have to be careful here because we are comparing a `string` (we don't really know if this is a decimal or hex encoding, but it seems that it tends to be a hex encoded)  with a `common.Hash`.

Reply:
This will also be improved once we update the monorepo import since the incoming depset type uses eth.ChainId instead of string for chainIds

Reply:
Yep